lesson_12/homework_12.py

```diff
@@ -1,10 +1,65 @@
+import csv
+import json
+import xml.etree.ElementTree as ET
+import logging
+from pathlib import Path
+
+my_path = Path("C:\\Hillel_lessons\\")
+
 # task 1
 """ Візміть два файли з теки
 ideas_for_test/work_with_csv
 порівняйте на наявність дублікатів
 результат запишіть у файл result_<your_second_name>.csv
 """
 
+def read_csv(file_path):
+    """
+    Reads a CSV file and returns the header and data as a tuple.
+    """
+    with open(file_path, 'r', newline='', encoding='utf-8') as file:
+        reader = csv.reader(file)
+        header = next(reader)
+        data = [row for row in reader]
+    return header, data
+
+
+def write_csv(file_path, header, data):
+    """
+    Writes data to a CSV file.
+    """
+    with open(file_path, 'w', newline='', encoding='utf-8') as file:
```

Comment:
```suggestion
    with open(file_path, 'w',  encoding='utf-8') as file:
```
не ясно нащо тут `newline=''`